api/graphql/schema.graphql

```diff
@@ -970,6 +970,7 @@ type Query {
   skills: [Skill]! @all
   genericJobTitle(id: UUID! @eq): GenericJobTitle @find
   genericJobTitles: [GenericJobTitle]! @all
+  team(id: UUID! @eq): Team @find @can(ability: "viewAny")
```

Comment:
For find queries like this, we should use the `view` policy so we can determine if the user can view this specific team. Right now, it will effectively be the same but when we implement the `view-team-team` and `view-own-team` permissions it will become important

Reply:
I doubt the `e2e` tests failing are related but I re-ran to be safe